### ISSUE TITLE
Build instructions for Windows

### ISSUE THREAD

**Issue by utkarsh-raj** [utkarsh-raj/umusic#26]
The project was built and tested on Linux environments. We need to build on Windows and note the process for people who prefer Windows.

**Comment by utkarsh-raj** [comment]
Will request @jai-dewani to look into this?

**Comment by jai-dewani** [comment]
Sure, you want me to try building the project on windows and fix any errors I find, right?

**Comment by utkarsh-raj** [comment]
Yes please, and we could discuss updating the Readme to now include the build instructions for Windows as well.

**Comment by jai-dewani** [comment]
I tried building it in widows and the current steps in README are working fine!
